library/std/src/sync/barrier.rs

```diff
@@ -1,3 +1,5 @@
+use core::panic::RefUnwindSafe;
```

Comment:
```suggestion
use crate::panic::RefUnwindSafe;
```
is a bit nicer since it doesn't break up the imports.